packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -100,23 +105,38 @@ export const buildBreadcrumbManager = (
         engine.state,
         facetId
       );
-      if (selectedValues.length > 0) {
-        breadcrumbs.push({
-          path: selectedValues,
-          deselect: () => {
-            dispatch(
-              executeDeselectAllCategoryFacetValues({
-                facetId,
-                numberOfValues: 5,
-              })
-            );
-          },
-        });
-      }
+      breadcrumbs.push({
+        field: engine.state.categoryFacetSet[facetId].field,
+        path: selectedValues,
+        deselect: () => {
+          dispatch(
+            executeDeselectAllCategoryFacetValues({
+              facetId,
+              numberOfValues: 5,
+            })
+          );
+        },
+      });
     });
     return breadcrumbs;
   }
 
+  function hasBreadcrumbs() {
```

Comment:
Should add a unit test for this method

Reply:
Thinking about this, does it really make senses to return breadcrumbs with empty values? These would never be displayed, right?
Maybe `getFacetBreadcrumbs()` and similar methods should return empty arrays when no breadcrumbs is active. Then you wouldn't need to filter out the ones with no values, or no path.

Reply:
I used this method to render (or not) the `Clear All Filters` button. `getFacetBreadcrumbs()` and similar methods return `null` if they are empty and they wouldn't be displayed. 

Generally, I think this method could be useful to clients so they don't have to implement their own.